fix incomplete bullets in "creating a tasklist" doc (#27420)
github/docs: content/issues/managing-your-tasks-with-tasklists/creating-a-tasklist.md

> Once you have started editing your tasklist Markdown, you can add new tasks by preceding each new task with `- [ ] ` and then adding either:

Reviewer: ```suggestion
Once you have started editing your tasklist Markdown, you can add new tasks by preceding each new task with `- [ ] ` and then adding one of the following:
```

Just adding this change to make the lead-in line to your main change make sense

- A draft task. Draft tasks are text that can later be converted into issues.

> - The full link to an issue or pull request.  For example, `https://github.com/octo-org/octo-repo/issues/45`.

Reviewer 2: Merged the 2nd and 3rd bullet points as otherwise the third reads weirdly when in conjunction with the "Once you have started..." sentence
```suggestion
- The full link to an issue or pull request.  For example, `https://github.com/octo-org/octo-repo/issues/45`. If an issue or pull request is in the same repository as your tasklist, you can use the #ISSUE-NUMBER shorthand syntax. For example, `#45`.
```

> - If an issue or pull request is in the same repository as your tasklist, you can use the #ISSUE-NUMBER shorthand syntax.  For example, `#45`.

Reviewer 2: ```suggestion
```
